grammar.ebnf

```diff
@@ -0,0 +1,499 @@
+(*
+ * THE CYPHER GRAPH QUERY LANGUAGE GRAMMAR (Draft, 2016-01-29)
+ *
+ * (c) Copyright 2015-2016 Neo Technology, Inc.
+ *
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *)
+
+Cypher = WS, AllOptions, WS, Statements ;
```

Comment:
I don't think 'Statements' is correct. Should probably be `AllOptions, WS, Statement`.